### ISSUE TITLE
PECO definition source appears twice in the browser, but is not in the file

### ISSUE THREAD

**Issue by cooperl09** [Planteome/amigo#35]
See: http://browser.planteome.org/amigo/term/EO:0007359#display-lineage-tab

http://dev.planteome.org/amigo/term/PECO:0007359#display-lineage-tab

Compare to stanza in file: 
[Term]
id: PECO:0007359
name: plant experimental condition
alt_id: EO:0007359
def: "A plant treatment (PECO:0001001) and/or study type (PECO:0007231) applied to a whole plant (PO:0000003), a plant structure (PO:00090119), and/or a plant population as part of an experiment to evaluate the plant response." [GR:pj, PO:cooperl]

Not sure why this is occurring in this ontology only?

**Comment by cmungall** [comment]
Hmm, this shouldn't be happening, there is no eo.owl in 
https://github.com/Planteome/planteome-core/blob/master/planteome.owl

**Comment by cmungall** [comment]
oh I see one URL is prod and another dev, I assume the prod one hasn't 
been updated since the EO->PECO switch

**Comment by elserj** [comment]
Correct, the actual issue is that the "source" references are there twice ( GR:pj, PO:cooperl, GR:pj, PO:cooperl).  Happens on both dev and live, so the issue is not new, but wasn't noticed before and apparently only happens with peco terms, although I haven't verified this.
